contents/blog/uh-i-like-my-job.md

```diff
@@ -0,0 +1,126 @@
+
+This gives me information to make better choices. I can evaluate the feedback and decide to lean further into places where my strengths are meshing with the team’s needs. I can make more intelligent tradeoffs between my ambitions and how my style impacts other people.
+
+Instead of being in the dark, I can navigate with confidence. Consistent feedback prevents resentments and poor direction from building up.
+
+## Fluid org chart
+
+The org chart is, in most places, treated like some holy truth carved in stone. It doesn’t readily change, except in calamitous, company-wide ruptures that change everyone’s lives overnight.
+
+Alternatively, you can build teams as [small squads](/handbook/company/small-teams), forming, dissolving, and combining talent as the needs of the business evolve.
+
+Every now and then I’ll look over my shoulder and see a new team has formed, cleaving off from previous work to start a new adventure.
+
+The flexibility has obvious advantages for the business, but for people, it’s an antidote to *stagnation.* The org chart evolves to suit the best fit of people on it, instead of trying to bend them into an imagined shape that doesn’t work. I can’t tell you how many people I’ve seen burn out because they couldn’t quite manage to become what someone randomly wrote down in a doc 18 months earlier.
+
+Instead, when we notice such a mismatch, a team’s *actual* strengths can be codified, and new teams can be chartered to fill any gaps. This is the luxury of a business that’s healthy and growing: slack in the system to also evolve.
+
+Even better, the org chart is descriptive, not prescriptive. It’s not a declaration of turf.
+
+I got to ship a new feature for our data pipelines product in my second week. *That’s not even my department*. So damn much fun.
+
+## Operational curiosity
+
+It is impossible to know everything in advance.
+
+A common startup failure mode is the tyrant founder who assumes he knows better than anyone else what must be done, what will work, where to go.
+
+But certainty is a trap for anyone trying something genuinely new. Curiosity, meanwhile, is the path to constant opportunity. You can’t know until you try. You can know even more if *everyone* is entitled to try.
+
+So everyone at PostHog is free to try, to fuck up, [to discover](/handbook/company/communication#top-tips-for-rfcs). If I have a strong opinion that disagrees even with a founder, I have the option to give it a swing anyway, if the decision is reversible.
+
+There’s [not an aggressive process](/handbook/values#4-trust-and-feedback-over-process) that dictates what I can do or how I have to do it. I have the trust of my team to make moves, and I’ll get feedback when I get it wrong. We course-correct into the right path through a sort of social eventual consistency.
```

Comment:
Is there a simpler framing than "social eventual consistency"? It's very eloquent, but somewhat opaque.

Reply:
It's a wink at a concept in distributed computing, for our developer audience, but let me know if it doesn't land:

https://en.wikipedia.org/wiki/Eventual_consistency

Reply:
Oh, ok. Just link to the wiki, then. Or maybe do a footnote. Either works.